launch/mongo-to-s3.yml

```diff
@@ -1,48 +1,48 @@
 run:
   type: docker
 env:
-- IL_URL
-- IL_USERNAME
-- IL_PASSWORD
-- IL_CONFIG
-- IL_USER_URL
-- IL_USER_USERNAME
-- IL_USER_PASSWORD
-- IL_USER_CONFIG
-- SIS_URL
-- SIS_USERNAME
-- SIS_PASSWORD
-- SIS_CONFIG
-- SIS_READ_URL
-- SIS_READ_USERNAME
-- SIS_READ_PASSWORD
-- SIS_READ_CONFIG
-- APP_SIS_URL
-- APP_SIS_USERNAME
-- APP_SIS_PASSWORD
-- APP_SIS_CONFIG
-- APP_SIS_READ_URL
-- APP_SIS_READ_USERNAME
-- APP_SIS_READ_PASSWORD
-- APP_SIS_READ_CONFIG
-- LEGACY_URL
-- LEGACY_USERNAME
-- LEGACY_PASSWORD
-- LEGACY_CONFIG
-- LEGACY_READ_URL
-- LEGACY_READ_USERNAME
-- LEGACY_READ_PASSWORD
-- LEGACY_READ_CONFIG
-- MISC_URL
-- MISC_USERNAME
-- MISC_PASSWORD
-- MISC_CONFIG
+  - IL_URL
+  - IL_USERNAME
+  - IL_PASSWORD
+  - IL_CONFIG
+  - IL_USER_URL
+  - IL_USER_USERNAME
+  - IL_USER_PASSWORD
+  - IL_USER_CONFIG
+  - SIS_URL
+  - SIS_USERNAME
+  - SIS_PASSWORD
+  - SIS_CONFIG
+  - SIS_READ_URL
+  - SIS_READ_USERNAME
+  - SIS_READ_PASSWORD
+  - SIS_READ_CONFIG
+  - APP_SIS_URL
+  - APP_SIS_USERNAME
+  - APP_SIS_PASSWORD
+  - APP_SIS_CONFIG
+  - APP_SIS_READ_URL
+  - APP_SIS_READ_USERNAME
+  - APP_SIS_READ_PASSWORD
+  - APP_SIS_READ_CONFIG
+  - LEGACY_URL
+  - LEGACY_USERNAME
+  - LEGACY_PASSWORD
+  - LEGACY_CONFIG
+  - LEGACY_READ_URL
+  - LEGACY_READ_USERNAME
+  - LEGACY_READ_PASSWORD
+  - LEGACY_READ_CONFIG
+  - MISC_URL
+  - MISC_USERNAME
+  - MISC_PASSWORD
+  - MISC_CONFIG
 resources:
-  cpu: 0.01
-  soft_mem_limit: 0.8
-  max_mem: 1.5
+  cpu: 1.0
+  soft_mem_limit: 1.6
+  max_mem: 3.0
```

Comment:
Mem seems to barely pass 100%, is there an advantage to doubling it?

Reply:
I just want to ensure that we have extra memory and we aren't pegged at 100%. Will reduce if we find it's not getting used.